internal/webhooks/machineset.go

```diff
@@ -50,23 +58,49 @@ func (webhook *MachineSet) SetupWebhookWithManager(mgr ctrl.Manager) error {
 // +kubebuilder:webhook:verbs=create;update,path=/mutate-cluster-x-k8s-io-v1beta1-machineset,mutating=true,failurePolicy=fail,matchPolicy=Equivalent,groups=cluster.x-k8s.io,resources=machinesets,versions=v1beta1,name=default.machineset.cluster.x-k8s.io,sideEffects=None,admissionReviewVersions=v1;v1beta1
 
 // MachineSet implements a validation and defaulting webhook for MachineSet.
-type MachineSet struct{}
+type MachineSet struct {
+	Decoder *admission.Decoder
```

Comment:
```suggestion
	decoder *admission.Decoder
```

Same in machinedeployment.go (we don't have to export it anymore as we are just setting it internally)

Reply:
✅ updated!